lib/elix/responders/lists.ex

```diff
@@ -11,7 +11,7 @@ defmodule Elix.Responders.Lists do
   show lists - Displays all lists
   """
   respond ~r/show lists\Z/i, msg do
-    reply(msg, render_items(List.all_names))
+    send(msg, render_items(List.all_names))
```

Comment:
I think this is reasonable. That is effectively null object. Really the decision is where does the defaulting happen. In this case it might make sense to push it down and have a `NullList` object that gets returned from the `List` module. Without typing it hard to say what I would like more.